### PR TITLE
Fix forward variable failMessage;

### PR DESCRIPTION
**Pull request by vonglasow**

### CODE REVIEW

**Comment by Hywan** [comment]
Agree with the fix.

**Comment by jubianchi** [comment]
@vonglasow :+1: can you add a changelog entry please :)

**Comment by jubianchi** [comment]
good to merge

**Comment by vonglasow** [comment]
@jubianchi added unit test

File: tests/units/classes/asserters/phpString.php [hunk 1]
```diff
@@ -589,4 +589,19 @@ public function testLength()
 					->isEqualTo(strlen($string))
 		;
 	}
+
+	public function testMatch()
+	{
+		$this
+			->if($asserter = $this->newTestedInstance)
+			->and($asserter->setWith('foo'))
```
Comment (jubianchi) [inline tests/units/classes/asserters/phpString.php]: please can you replace the hardcoded `foo` by a call to `uniqid()`

File: tests/units/classes/asserters/phpString.php [hunk 2]
```diff
@@ -589,4 +589,19 @@ public function testLength()
...
+			->and($failMessage = uniqid())
+			->then
+				->exception(function() use ($asserter, $failMessage) {
+					$asserter->match('/bar/', $failMessage);
```
Comment (jubianchi) [inline tests/units/classes/asserters/phpString.php]: same thing here: replace the hardcoded value with a random one :)

**Pull request merged**